HMCL/src/main/java/org/jackhuang/hmcl/ui/LogWindow.java

```diff
@@ -353,15 +381,25 @@ protected void updateItem(Log item, boolean empty) {
                 autoScrollCheckBox.setSelected(true);
                 control.autoScroll.bind(autoScrollCheckBox.selectedProperty());
 
+                JFXButton exportLogsButton = new JFXButton(i18n("button.export"));
+                exportLogsButton.setOnMouseClicked(e -> getSkinnable().onExportLogs());
```

Comment:
```suggestion
                exportLogsButton.setOnAction(e -> getSkinnable().onExportLogs());
```

`setOnMouseClicked` 没法用键盘触发。当然你不改也行，过几天我会把整个仓库里的 `setOnMouseClicked` 全部清理一遍。